### PR TITLE
Data from DB to front end

### PR DESCRIPTION
**Pull request by ChrisThompsonK**

### CODE REVIEW

File: ExpressAPI/db.js [hunk 1]
```diff
@@ -3,48 +3,52 @@ const log4js = require('log4js');
 const logger = log4js.getLogger();
 logger.level = 'debug';
 
+
```
Comment (micalexis) [inline ExpressAPI/db.js]: unnecessary white space, remove

File: ExpressAPI/db.js [hunk 2]
```diff
@@ -60,3 +64,20 @@ exports.getCompetencies = function(callback) {
         }
     )
 }
+
+exports.getTrainingPerBand = function (bandID, callback) {
+    db.query(
+        /*
```
Comment (micalexis) [inline ExpressAPI/db.js]: remove comment if not required

File: kjourney/src/app/bands/bands.component.ts [hunk 3]
```diff
@@ -1,24 +1,28 @@
...
+  public trainingPerBand: TrainingPerBand[]
   public competencies: Competency[]
 
   constructor(data: DataService) {
-    this.data = data;
+    this.data = data
   }
 
   async ngOnInit() {
+    this.trainingPerBand = await this.data.getTrainingPerBand(1);
```
Comment (micalexis) [inline kjourney/src/app/bands/bands.component.ts]: change to use Observables not Promises

File: kjourney/src/app/data.service.ts [hunk 4]
```diff
@@ -19,4 +19,8 @@ export class DataService {
   public getCompetencies(): Observable<Competency[]> {
     return this.http.get<Competency[]>('api/competencies');
   }
+
+  public getTrainingPerBand(bandID): Promise<TrainingPerBand[]> {
```
Comment (micalexis) [inline kjourney/src/app/data.service.ts]: change to use Observables not Promises

**Review by micalexis — CHANGES_REQUESTED** [review]
apply changes and squash commits

**Review by Conormc98 — APPROVED** [review]

**Review by micalexis — APPROVED** [review]

**Pull request merged**